modules/tasty-lib/src/main/scala/scala/build/tastylib/TastyVersions.scala

```diff
@@ -1,15 +1,13 @@
 package scala.build.tastylib
 
+import scala.build.tastylib.internal.Constants
+
 object TastyVersions {
 
   // Every time tasty version is updated, please update LatestSupportedScala as well!
-  final val MajorVersion: Int        = 28
-  final val MinorVersion: Int        = 1
-  final val ExperimentalVersion: Int = 0
-
   object LatestSupportedScala {
     final val MajorVersion: Int = 3
```

Comment:
it's not like Scala 4 is coming out anytime soon, but this could be derived from `latestSupportedScala` as well, IMO
```suggestion
    final val MajorVersion: Int = Constants.latestSupportedScala.split('.').head.toInt
```